internal/form/album_test.go

```diff
@@ -35,6 +41,9 @@ func TestNewAlbum(t *testing.T) {
 		assert.Equal(t, "test notes", r.AlbumNotes)
 		assert.Equal(t, "newest", r.AlbumOrder)
 		assert.Equal(t, "default", r.AlbumTemplate)
+		assert.Equal(t, 2050, r.AlbumYear)
+		assert.Equal(t, 20, r.AlbumMonth)
+		assert.Equal(t, 37, r.AlbumDay)
```

Comment:
What is day 37 of month 20?

Reply:
Just wanted to highlight that there are no sanity checks for the month and day values. If its bothering you, I can set them to something more plausible.

Reply:
It will probably be rejected when persisting date in the database, but that's not a concern in this context. Thanks for your answer.

Reply:
Well, surprisingly, it isn't - I did few tests and have few albums with `month > 12` right now.